coalib/results/result_actions/ApplyPatchAction.py

```diff
@@ -30,7 +30,7 @@ def apply(self,
               file_diff_dict,
               no_orig: bool=False):
         """
-        Apply the patch automatically.
+        (a)pply the patch.
```

Comment:
In my opinion, if we were to use this, you should still capitalize the first letter.